app/src/hooks/useSymbol.tsx

```diff
@@ -0,0 +1,14 @@
+import { getNativeAsset, getWrapToken } from '../util/networks'
```

Comment:
I think this is a candidate for a util function rather than a hook.
Hooks typically modify the state in some way, this is a stateless function that takes an input and returns an output.

Reply:
for sure this would def make a great util function but it depends on another hook `useConnectedWeb3Context` so just keeping hookish things grouped together. It would be great if I could just pass networkId down as a param but this hook is used in places where networkId is not available.